### ISSUE TITLE
Tensorboard not showing results 

### ISSUE THREAD

**Issue by scrunguss** [tensorflow/tensorboard#5576]
### Diagnostics

<details>
<summary>Diagnostics output</summary>

``````
--- check: autoidentify
INFO: diagnose_tensorboard.py version e43767ef2b648d0d5d57c00f38ccbd38390e38da

--- check: general
INFO: sys.version_info: sys.version_info(major=3, minor=9, micro=5, releaselevel='final', serial=0)
INFO: os.name: posix
INFO: os.uname(): posix.uname_result(sysname='Linux', nodename='scrungus-pc', release='5.11.0-49-generic', version='#55-Ubuntu SMP Wed Jan 12 17:36:34 UTC 2022', machine='x86_64')
INFO: sys.getwindowsversion(): N/A

--- check: package_management
INFO: has conda-meta: False
INFO: $VIRTUAL_ENV: None

--- check: installed_packages
INFO: installed: tensorboard==2.8.0
INFO: installed: tensorflow==2.7.1
INFO: installed: tensorflow-estimator==2.7.0
INFO: installed: tensorboard-data-server==0.6.1

--- check: tensorboard_python_version
INFO: tensorboard.version.VERSION: '2.8.0'

--- check: tensorflow_python_version
2022-02-22 16:24:30.047026: W tensorflow/stream_executor/platform/default/dso_loader.cc:64] Could not load dynamic library 'libcudart.so.11.0'; dlerror: libcudart.so.11.0: cannot open shared object file: No such file or directory
2022-02-22 16:24:30.047052: I tensorflow/stream_executor/cuda/cudart_stub.cc:29] Ignore above cudart dlerror if you do not have a GPU set up on your machine.
INFO: tensorflow.__version__: '2.7.1'
INFO: tensorflow.__git_version__: 'v2.7.0-217-g2a0f59ecfe6'

--- check: tensorboard_data_server_version
INFO: data server binary: '/home/scrungus/.local/lib/python3.9/site-packages/tensorboard_data_server/bin/server'
INFO: data server binary version: b'rustboard 0.6.1'

--- check: tensorboard_binary_path
INFO: which tensorboard: b'/home/scrungus/.local/bin/tensorboard\n'

--- check: addrinfos
socket.has_ipv6 = True
socket.AF_UNSPEC = <AddressFamily.AF_UNSPEC: 0>
socket.SOCK_STREAM = <SocketKind.SOCK_STREAM: 1>
socket.AI_ADDRCONFIG = <AddressInfo.AI_ADDRCONFIG: 32>
socket.AI_PASSIVE = <AddressInfo.AI_PASSIVE: 1>
Loopback flags: <AddressInfo.AI_ADDRCONFIG: 32>
Loopback infos: [(<AddressFamily.AF_INET6: 10>, <SocketKind.SOCK_STREAM: 1>, 6, '', ('::1', 0, 0, 0)), (<AddressFamily.AF_INET: 2>, <SocketKind.SOCK_STREAM: 1>, 6, '', ('127.0.0.1', 0))]
Wildcard flags: <AddressInfo.AI_PASSIVE: 1>
Wildcard infos: [(<AddressFamily.AF_INET: 2>, <SocketKind.SOCK_STREAM: 1>, 6, '', ('0.0.0.0', 0)), (<AddressFamily.AF_INET6: 10>, <SocketKind.SOCK_STREAM: 1>, 6, '', ('::', 0, 0, 0))]

--- check: readable_fqdn
INFO: socket.getfqdn(): 'scrungus-pc'

--- check: stat_tensorboardinfo
INFO: directory: /tmp/.tensorboard-info
INFO: os.stat(...): os.stat_result(st_mode=16895, st_ino=6563770, st_dev=66309, st_nlink=2, st_uid=1000, st_gid=1000, st_size=4096, st_atime=1645544353, st_mtime=1645547008, st_ctime=1645547008)
INFO: mode: 0o40777

--- check: source_trees_without_genfiles
INFO: tensorboard_roots (1): ['/home/scrungus/.local/lib/python3.9/site-packages']; bad_roots (0): []

--- check: full_pip_freeze
INFO: pip freeze --all:
absl-py==1.0.0
aiohttp==3.8.1
aiohttp-socks==0.7.1
aiorpcX==0.18.7
aiosignal==1.2.0
apsw==3.34.0.post1
apt-xapian-index==0.49
apturl==0.5.2
argon2-cffi==18.3.0
astunparse==1.6.3
async-timeout==4.0.2
attrs==21.4.0
backcall==0.2.0
backports.entry-points-selectable==1.1.1
beautifulsoup4==4.9.3
bitstring==3.1.9
bleach==3.2.1
blinker==1.4
Brlapi==0.8.2
cachetools==5.0.0
certifi==2020.6.20
chardet==4.0.0
charset-normalizer==2.0.11
cliapp==1.20180812.1
click==7.1.2
cloudpickle==1.6.0
cmdtest==0.32+git
colorama==0.4.4
cryptography==3.3.2
css-parser==1.0.6
cssselect==1.1.0
cssutils==1.0.2
cupshelpers==1.0
cycler==0.11.0
dbus-python==1.2.16
decorator==5.1.1
defer==1.0.6
defusedxml==0.6.0
distlib==0.3.4
distro==1.5.0
distro-info==1.0
dm-reverb==0.6.1
dm-tree==0.1.6
dnspython==2.0.0
docopt==0.6.2
ecdsa==0.16.1
Electrum @ https://download.electrum.org/4.1.5/Electrum-4.1.5.tar.gz
entrypoints==0.3
feedparser==5.2.1
filelock==3.4.0
flatbuffers==2.0
fonttools==4.29.1
frozenlist==1.3.0
fsspec==2022.1.0
future==0.18.2
gast==0.4.0
gin-config==0.5.0
google-auth==2.6.0
google-auth-oauthlib==0.4.6
google-pasta==0.2.0
gpg===1.14.0-unknown
grpcio==1.43.0
gym==0.19.0
-e git+https://github.com/scrunguss/DQNAgent.git@abc41191710efbc6d894ec9979a197e9382f638c#egg=gym_go
gyp==0.1
h5py==3.6.0
helpdev==0.7.1
html2text==2020.1.16
html5-parser==0.4.9
html5lib==1.1
httplib2==0.18.1
idna==2.10
ifaddr==0.1.7
imageio==2.4.0
importlib-metadata==1.6.0
ipykernel==5.4.3
ipython==7.20.0
ipython_genutils==0.2.0
ipywidgets==6.0.0
jedi==0.18.0
jeepney==0.6.0
Jinja2==2.11.2
joblib==1.1.0
jsonschema==3.2.0
jupyter-client==6.1.11
jupyter-console==6.2.0
jupyter-core==4.7.1
jupyterthemes==0.20.0
kazam==1.4.5
keras==2.7.0
Keras-Preprocessing==1.1.2
keyring==22.2.0
kiwisolver==1.3.2
language-selector==0.1
launchpadlib==1.10.13
lazr.restfulclient==0.14.2
lazr.uri==1.0.5
lesscpy==0.15.0
libclang==13.0.0
louis==3.16.0
lxml==4.6.3
macaroonbakery==1.3.1
Markdown==3.3.4
MarkupSafe==1.1.1
matplotlib==3.5.1
mechanize==0.4.5
mistune==0.8.4
more-itertools==4.2.0
msgpack==1.0.0
multidict==6.0.2
nbconvert==5.6.1
nbformat==5.1.2
netifaces==0.10.9
notebook==6.2.0
numpy==1.21.3
oauthlib==3.1.0
olefile==0.46
opencv-python==4.5.5.62
opt-einsum==3.3.0
packaging==20.9
pandas==1.4.1
pandocfilters==1.4.3
parso==0.8.1
pexpect==4.8.0
pickleshare==0.7.5
Pillow==8.1.2
pip==22.0.3
pipenv==2021.11.23
pipreq==0.4
pipreqs==0.4.11
platformdirs==2.4.0
ply==3.11
portpicker==1.5.0
prometheus-client==0.9.0
prompt-toolkit==3.0.14
protobuf==3.12.4
psutil==5.9.0
py7zr==0.0.0
pyasn1==0.4.8
pyasn1-modules==0.2.8
pycairo==1.16.2
pychm==0.8.6
pycryptodomex==3.9.7
pycups==2.0.1
pyDeprecate==0.3.1
pyglet==1.5.21
Pygments==2.7.1
PyGObject==3.38.0
PyJWT==1.7.1
pymacaroons==0.13.0
PyNaCl==1.4.0
pyparsing==2.4.7
PyQt5==5.15.4
PyQt5-sip==12.8.1
PyQtWebEngine==5.15.4
pyRFC3339==1.1
pyrsistent==0.15.5
PySocks==1.7.1
python-apt===2.2.0-ubuntu0.21.04.1
python-dateutil==2.8.1
python-debian==0.1.39
python-socks==2.0.3
pytorch-lightning==1.5.10
pytz==2021.1
PyVirtualDisplay==3.0
pyxdg==0.27
PyYAML==5.3.1
pyzmq==20.0.0
QDarkStyle==2.8.1
qrcode==7.3.1
QtPy==2.0.1
regex==2020.11.13
reportlab==3.5.66
repoze.lru==0.7
requests==2.25.1
requests-oauthlib==1.3.1
Routes==2.5.1
rsa==4.8
scikit-learn==1.0.2
scipy==1.8.0
SecretStorage==3.3.1
selenium==3.141.0
Send2Trash==1.6.0b1
setuptools==59.5.0
simplejson==3.17.2
six==1.15.0
sklearn==0.0
soupsieve==2.2
stable-baselines3==1.4.0
systemd-python==234
tabulate==0.8.9
tensorboard==2.8.0
tensorboard-data-server==0.6.1
tensorboard-plugin-wit==1.8.1
tensorflow==2.7.1
tensorflow-estimator==2.7.0
tensorflow-io-gcs-filesystem==0.24.0
tensorflow-probability==0.15.0
termcolor==1.1.0
terminado==0.9.2
testpath==0.4.4
texttable==1.6.3
tf-agents==0.11.0
threadpoolctl==3.1.0
torbrowser-launcher==0.3.3
torch==1.8.1
torchmetrics==0.7.2
torchvision==0.11.3
tornado==6.1
tqdm==4.62.3
traitlets==5.0.5
ttystatus==0.38
typing_extensions==4.1.0
ubuntu-drivers-common==0.0.0
unattended-upgrades==0.1
urllib3==1.26.2
usb-creator==0.3.7
virtualenv==20.10.0
virtualenv-clone==0.5.7
wadllib==1.3.5
wcwidth==0.1.9
webencodings==0.5.1
WebOb==1.8.6
Werkzeug==2.0.3
wheel==0.34.2
widgetsnbextension==2.0.0
wrapt==1.13.3
xdg==5
xkit==0.0.0
yarg==0.1.9
yarl==1.7.2
zeroconf==0.26.1
zipp==1.0.0

``````

</details>

## Issue description
I can't seem to get tensorboard to display any output. I've trained a model successfully using PyTorch Lightning. Here is the structure of the log folder:

```
.
├── version_0
│   ├── checkpoints
│   │   └── epoch=2-step=191.ckpt
│   └── hparams.yaml
├── version_1
│   ├── checkpoints
│   │   └── epoch=2-step=191.ckpt
│   └── hparams.yaml
└── version_2
    ├── checkpoints
    │   └── epoch=2-step=191.ckpt
    └── hparams.yaml

6 directories, 6 files
```
And logging is done in `training_step()` like : 

```

 self.log("total_reward", torch.tensor(self.total_reward).to(device), on_step=True, on_epoch=True, prog_bar=True, logger=True)
        self.log("reward", torch.tensor(reward).to(device), on_step=True, on_epoch=True, prog_bar=True, logger=True)
        self.log("train_loss", loss, on_step=True, on_epoch=True, prog_bar=True, logger=True)

log = {
            "total_reward": torch.tensor(self.total_reward).to(device),
            "reward": torch.tensor(reward).to(device),
            "train_loss": loss,
        }
        status = {
            "steps": torch.tensor(self.global_step).to(device),
            "total_reward": torch.tensor(self.total_reward).to(device),
        }

        return OrderedDict({"loss": loss, "log": log, "progress_bar": status})
```

Yet when I run tensorboard: 

```
~/Documents/dissertation/dqn/GymGo$ tensorboard --logdir lightning_logs #lightning_logs folder is here
2022-02-22 16:28:27.944967: W tensorflow/stream_executor/platform/default/dso_loader.cc:64] Could not load dynamic library 'libcudart.so.11.0'; dlerror: libcudart.so.11.0: cannot open shared object file: No such file or directory
2022-02-22 16:28:27.944991: I tensorflow/stream_executor/cuda/cudart_stub.cc:29] Ignore above cudart dlerror if you do not have a GPU set up on your machine.
2022-02-22 16:28:29.035412: W tensorflow/stream_executor/platform/default/dso_loader.cc:64] Could not load dynamic library 'libcuda.so.1'; dlerror: libcuda.so.1: cannot open shared object file: No such file or directory
2022-02-22 16:28:29.035437: W tensorflow/stream_executor/cuda/cuda_driver.cc:269] failed call to cuInit: UNKNOWN ERROR (303)
2022-02-22 16:28:29.035450: I tensorflow/stream_executor/cuda/cuda_diagnostics.cc:156] kernel driver does not appear to be running on this host (scrungus-pc): /proc/driver/nvidia/version does not exist

```
I get the message on tensorboard:  `No dashboards are active for the current data set.`. What am I doing wrong?
